docs/autojustice.rst

```diff
@@ -19,22 +19,22 @@ Usage
 Options
 -------
 
-``-j``, ``--jailcitizen``
+``-j true|false``, ``--jailcitizen true|false``
```

Comment:
the usual style here is to only document the parameters for the long form to avoid duplication